Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocIndexColumnRenderer.cs

```diff
@@ -100,16 +102,16 @@ public void RenderTitleCell (BocListRenderingContext renderingContext)
       if (_renderingFeatures.EnableDiagnosticMetadata)
       {
         var columnTitle = renderingContext.Control.IndexColumnTitle;
-        if (!string.IsNullOrEmpty (columnTitle))
-          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
+        if (!columnTitle.IsEmpty)
+          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle.GetValue()));
```

Comment:
```suggestion
          renderingContext.Writer.AddAttribute (DiagnosticMetadataAttributes.Content, HtmlUtility.StripHtmlTags (columnTitle));
```

Because I was stupid...